variables.tf

```
@@ -27,6 +27,12 @@ variable "dead_letter_target_arn" {
   description = "The ARN of an SNS topic or SQS queue to notify when an invocation fails"
 }
 
+variable "enable_all_egress_rule" {
+  type        = bool
+  default     = false
```

```suggestion
  default     = true
```

This should be set to `true` to maintain compatibility, otherwise we're introducing a breaking change for others. Those that don't want to enable it can then opt out.

---

@shoekstra this default setting was specifically requested by @marwinbaumannsbp to set a secure default, as that is what mcaf is supposed to be. he suggested raising the version to 0.3.0 to indicate a major change. I understand your suggestion, but I think setting a secure default makes more sense in this case. Can you discuss with @marwinbaumannsbp how to proceed?